src/v2/Apps/Consign/Routes/MarketingLanding/Components/SellWithArtsy.tsx

```diff
@@ -1,25 +1,43 @@
-import * as React from "react";
+import * as React from "react"
 import { Box, Button, Flex, Image, Link, Text, color } from "@artsy/palette"
 import { SectionContainer } from "./SectionContainer"
+import { useAnalyticsContext } from "v2/System"
 import { Media } from "v2/Utils/Responsive"
 import { useTracking } from "react-tracking"
-import { ContextModule, OwnerType, clickedAppDownload } from "@artsy/cohesion"
+import {
+  ActionType,
+  ContextModule,
+  OwnerType,
+  ClickedAppDownload,
+  PageOwnerType,
+} from "@artsy/cohesion"
 
 const DOWNLOAD_URL =
   "https://apps.apple.com/us/app/artsy-buy-sell-original-art/id703796080"
 
+interface AnalyticsOptions {
+  context_module: ContextModule
+  destination_path: string
+  context_page_owner_id?: string
+  context_page_owner_slug?: string
+  context_page_owner_type: PageOwnerType
+}
+
 export const SellWithArtsy: React.FC = () => {
   const tracking = useTracking()
 
+  const { contextPageOwnerId, contextPageOwnerSlug } = useAnalyticsContext()
+
+  const options: AnalyticsOptions = {
+    context_module: ContextModule.sellFooter,
+    destination_path: DOWNLOAD_URL,
+    context_page_owner_type: OwnerType.consign,
+    context_page_owner_slug: contextPageOwnerSlug,
+    context_page_owner_id: contextPageOwnerId,
```

Comment:
Is this correct?
The only reason why I'm setting `context_page_owner_slug` and `context_page_owner_id` is that I saw these were already being generated when first writing the specs for this component, with the previous implementation.

Therefore, to have this available I'm setting these variables as a result of `useAnalyticsContext()` on L29
I did this because it's what we also do on [DownloadAppBadge](https://github.com/artsy/force/blob/c4e5f1cbdb805d8157ea9bdd5f8880f9841f9481/src/v2/Components/DownloadAppBadge.tsx#L46-L50) component. But I thought it would be good to confirm if this is correct (as we don't need to do this here with the current implementation)

Reply:
I'd say either way is fine; separate branch is perhaps a little "cleaner" from a code history perspective, but it's a minor enough change that I don't feel strongly about it :)